src/components/screens/DietaEspecial/Relatorio/componentes/EscolaCancelaDietaEspecial.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from "react";
+
+import Botao from "../../../../Shareable/Botao";
+import {
+  BUTTON_TYPE,
+  BUTTON_STYLE
+} from "../../../../Shareable/Botao/constants";
+import ModalCancelaDietaEspecial from "./ModalCancelaDietaEspecial";
+
+export default class EscolaCancelaDietaEspecial extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false
+    };
+    this.closeModal = this.closeModal.bind(this);
+    this.openModal = this.openModal.bind(this);
+  }
+
+  openModal() {
+    this.setState({ showModal: true });
+  }
+
+  closeModal() {
+    this.setState({ showModal: false });
+  }
+
+  render() {
+    const { showModal } = this.state;
+    const { onCancelar } = this.props;
+    return (
+      <>
+        <ModalCancelaDietaEspecial
+          showModal={showModal}
+          closeModal={this.closeModal}
```

Comment:
Sugestão: Aqui ficou estranho pq um é um booleano e o outro é uma função mas a nomenclatura está igual. Uma soluçáo seria nomear a funçáo this.onCloseModal